astrospec/spectrum1d.py

```diff
@@ -8,10 +9,18 @@
 
 from warnings import warn
 
+from .wcs import WCSTable
+
 
 class Spectrum1D(NDSlicingMixin, NDIOMixin, NDData):
-    """A 1 dimensionan spectrum.
-    
+    """A 1-dimensional spectrum.
+
+    Attributes
+    ----------
+
+    Methoods
+    --------
+
```

Comment:
We should decide this now I suppose. Astropy is rather inconsistent with this. I prefer numpy style doc strings to google style. You are correct that introspection gives us what we need in the IPython shell and elsewhere. We need not list "Methods" (I can't believe I miss spelled that). But I think it is helpful to list Attributes (the major ones that are import to the class. I know you can tab complete to get everything that Spectrum1D.* is possible, but I think it's still a good idea to briefly state the major attributes and what they are for.